runtime/doc/editing.txt

```diff
@@ -1040,7 +1040,7 @@ if the system allows it (the directory must be writable).
 							*write-fail*
 If the writing of the new file fails, you have to be careful not to lose
 your changes AND the original file.  If there is no backup file and writing
-the new file failed, you have already lost the original file!  DON'T EXIT VIM
+the new file failed, you have already lost the original file!  DON'T EXIT Vim
```

Comment:
In contexts where the sentence is all upper-case, I think it's better to write VIM uppper-case too.